utils/lister.hh

```diff
@@ -202,6 +202,10 @@ public:
 
     future<std::optional<directory_entry>> get() override;
     future<> close() noexcept override;
+
+    file& directory() {
```

Comment:
Why is it needed with seastar update?

Reply:
Sorry, bad rebase.
It's a remnant of a more intrusive change the I partially reverted locally.
Will drop